testdata/project-v4/Makefile

```diff
@@ -119,10 +119,6 @@ docker-buildx: ## Build and push docker image for the manager for cross-platform
 build-installer: manifests generate kustomize ## Generate a consolidated YAML with CRDs and deployment.
 	mkdir -p dist
 	echo "---" > dist/install.yaml # Clean previous content
```

Comment:
you don't need this line with suggestion bottom.

Reply:
agreed, will remove in next commit. Thanks!